resources/deployments/controller/controller_deployments.go

```diff
@@ -65,7 +65,7 @@ func (d *DeploymentsController) PostDeployment(w rest.ResponseWriter, r *rest.Re
 		return
 	}
 
-	d.view.RenderSuccessPost(w, r, id)
+	d.view.RenderSuccessPost(w, r, "./deployments/"+id)
```

Comment:
adding header and it's content is self contained in view, why not modifying that instead?